Makefile

```diff
@@ -6,4 +6,14 @@ image:
 	docker build --rm -t $(IMAGE_NAME) .
 
 test: image
-	docker run --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && bundle exec rake"
+	docker run --rm \
+	  --env CIRCLECI \
+	  --env CIRCLE_BUILD_NUM \
+	  --env CIRCLE_BRANCH \
+	  --env CIRCLE_SHA1 \
+		--env CODECLIMATE_REPO_TOKEN \
+	  --net host \
```

Comment:
oof this is what I get for copying lines from another repo's Makefile 😄. I assumed `--net host` was to allow the container to submit the test results to CC. I'll try removing and see if it fails to report it.